python/private/py_wheel.bzl

```diff
@@ -473,6 +479,28 @@ def _py_wheel_impl(ctx):
             filename + ";" + target_files[0].path,
         )
 
+    for target, filename in ctx.attr.data_files.items():
+        target_files = target.files.to_list()
+        if len(target_files) != 1:
+            fail(
+                "Multi-file target listed in data_files %s",
+                filename,
+            )
+
+        if not filename.startswith(ALLOWED_DATA_FILE_PREFIX):
```

Comment:
We should probably check that / follows straight after the allowed prefix value.

Maybe something like `filename.partition("/")[0] not in ALLOWED_DATA_FILE_PREFIX)` would do the trick?

Reply:
Fixed and added another unit test